.travis.yml

```diff
@@ -31,6 +31,11 @@ language: python
 python:
   - 2.7
 
+env:
+  - PYTEST="tests/test_views.py"
+  - PYTEST="tests/test_workflows.py"
+  - PYTEST="--ignore tests/test_views.py --ignore tests/test_workflows.py"
```

Comment:
BTW: sure. This was just a quick thing to gain 5 minutes in all the next testing. On the other hand, Travis testing is as slow as the slowest of its matrix component, so if you break up unit testing and integration testing only, the bottle neck will still be integration. We would still need to break it up a bit.